test/test_dynamic_shapes.py

```diff
@@ -1770,6 +1772,36 @@ def test_dim_constraints_solve_full(self):
             "dynamic_dim(L['x9'], 1) == dynamic_dim(L['x6'], 1)",
         })
 
+        def dummy_f(x0, x1, x2, x3, x4, x5, x6, x7, x8, x9, x11, x12):
+            pass
+
+        action_code = dim_constraints.prettify_results(inspect.signature(dummy_f))
+        static_code, dynamic_code = re.findall(r"```(.*?)```", action_code, re.DOTALL)
+        expected_static = '''
+def static_constraints(x0, x1, x2, x3, x4, x5, x6, x7, x8, x9, x11, x12):
```

Comment:
This function doesn't do anything. One idea would be to join the constraints with `and` (still on separate lines). Then perhaps the user can call it to check specialized inputs.

While you're at it, call this `specializations` and call the other `specify_constraints`, since the API calls the result `constraints`.

Reply:
yeah, sounds good to me. For specializations call, it seems to possibly contain assignment operator "=": https://github.com/pytorch/pytorch/blob/main/torch/fx/experimental/symbolic_shapes.py#L1638.  Should we just turn it into "==" and  in the _static_results?